### ISSUE TITLE
Feature request: return block references

### ISSUE THREAD

**Issue by faurehu** [galihlprakoso/logseq-plugin-assistseq-ai-assistant#6]
I would like that the RAG also returned the references of the blocks so that I can go to the place in my writings where the answer to my question lies.

**Comment by galihlprakoso** [comment]
https://github.com/galihlprakoso/logseq-plugin-assistseq-ai-assistant/releases/tag/v2.4.0

**Comment by faurehu** [comment]
Thank you!

I installed it and it only shows me the spinner with "Loading your LogSeq documents..."

**Comment by galihlprakoso** [comment]
> Thank you!
> 
> I installed it and it only shows me the spinner with "Loading your LogSeq documents..."

What kind of page you currently opened?

<img width="610" alt="Screenshot 2024-09-16 at 07 10 58" src="https://github.com/user-attachments/assets/de4026aa-ad95-44e6-95a3-37730c2fe076">

And also try to reduce this number. Tell me if it fixed the issue. @faurehu

**Comment by faurehu** [comment]
Thanks for your help! It worked for a minute, without me changing any settings, and then I closed the app and now it's not working again. It won't work even by setting maxRecursionDepth to 5 (previously it was 3).

Also, when it was working, there were a few thing that were not working. First, I wasn't able to see the block id in order to go to the source. Furthermore, when I clicked on copy to clipboard, it was pasting something else (it seemed to be the code of the extension).
